ui/app/mainui/activitycenter/views/ActivityNotificationCommunityShareAddresses.qml

```diff
@@ -0,0 +1,81 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+import shared 1.0
+import shared.panels 1.0
+import utils 1.0
+
+import "../panels"
+import "../popups"
+import "../stores"
+
+ActivityNotificationBase {
+    id: root
+
+    required property string communityName
+    required property string communityColor
+    required property string communityImage
+
+    signal openCommunityClicked
+    signal openShareAccountsClicked
+
+    bodyComponent: RowLayout {
+        spacing: 8
+
+        StatusSmartIdenticon {
+            name: root.communityName
+            Layout.preferredWidth: 40
+            Layout.preferredHeight: 40
+            Layout.alignment: Qt.AlignTop
+            Layout.leftMargin: Style.current.padding
+            Layout.topMargin: 2
+
+            asset {
+                width: 24
+                height: width
+                name: root.communityImage
+                color: root.communityColor
+                bgWidth: 40
+                bgHeight: 40
+            }
+        }
+
+        ColumnLayout {
+            spacing: 2
+            Layout.alignment: Qt.AlignTop
+            Layout.fillWidth: true
+
+            StatusMessageHeader {
+                displayNameLabel.text: qsTr("Share accounts to %1 Community required").arg(root.communityName)
```

Comment:
```suggestion
                displayNameLabel.text: qsTr("Share accounts with %1 Community required").arg(root.communityName)
```

Reply:
Switching it up a little, because you don,t really share it with the community, just the owner.
```suggestion
                displayNameLabel.text: qsTr("%1 requires you to share your Accounts").arg(root.communityName)
```